portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/snippets/skipNav-snippet.vm

```diff
@@ -54,6 +54,7 @@
         </li>
     </ul>
 </nav>
+<div id="select-site-sidebar" class="offcanvas offcanvas-start" tabindex="-1" aria-labelledby="select-site-sidebar-label"></div>
```

Comment:
does this result in the same id being used two times? it is also used in moresites.vm

Reply:
the issue appears to be that this assistive skipNav is pointing to `aria-controls="select-site-sidebar"` and the `id=select-site-sidebar` is not present on the gateway because the user is not logged in.

the correct fix here would be to edit this skipNav to not display the whole chunk unless the user is logged in. it should be done the same way as in moresites.vm:

`#if (${userIsLoggedIn})`

Reply:
Thanks for your advice, Sam. I tested the issue with your code and I don't see the aXe error when the user is logged in. 

However, aXe still flags an error on the page before the user logs in. Do you think I should add the div with id="select-site-sidebar" in my code?

<img width="1440" alt="Screen Shot 2024-10-24 at 1 44 08 PM" src="https://github.com/user-attachments/assets/2ba9a9c8-7d4f-4d8e-8706-608d7a09aebd">

Reply:
> I tested the issue with your code and I don't see the aXe error when the user is logged in.

Sorry did you make a code change? I don't see it. No, I don't think adding a new div is the correct fix.